### PR TITLE
convert celsius to fahrenheit

### LINKED ISSUE #30: converter-react with typescript
converting Celsius to Fahrenheit and viseversa

### PR DESCRIPTION
**Pull request by schagdeh**
close #30

### CODE REVIEW

File: 03-react/04-converter/src/main.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+			/>
+			{" + "}
+			<input
+				type="number"
+				value={b}
+				placeholder="°C"
+				onChange={(e) => cTof(e)}
+			/>
+			{/* {" = "}
+			<span>{toInt(a) + toInt(b)}</span> */}
```
Comment (mbovel) [inline 03-react/04-converter/src/main.tsx:57]: You could remove that.

File: 03-react/04-converter/src/main.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+const App = (): JSX.Element => {
+	const [a, setA] = React.useState("");
+	const [b, setB] = React.useState("");
+	console.log(a);
+	
+	
+	const fToc = (e: React.ChangeEvent<HTMLInputElement>) => {
+		setA(e.target.value)
+		const celsius = ((toInt(e.target.value) - 32) * 5/9).toFixed(2).toString()
```
Comment (mbovel) [inline 03-react/04-converter/src/main.tsx:28]: `toFixed(2)` already returns a string, therefore you do not need the following `toString()`.

File: 03-react/04-converter/src/main.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+		setB(celsius)
+		e.target.value === '' && setB('')
+		 return 
+	}
+	const cTof = (e: React.ChangeEvent<HTMLInputElement>) => {
+		setB(e.target.value)
+		const celsius = ((toInt(e.target.value) * 9/5) + 32).toFixed(2).toString()
```
Comment (mbovel) [inline 03-react/04-converter/src/main.tsx:35]: I would change the name of this variable, these are probably Fahrenheit degrees 😉 .

**Review by mbovel — APPROVED** [review]
Good work!

See my 3 minor comments.

File: 03-react/04-converter/src/main.tsx (hunk 3 above)
Comment (schagdeh) [reply 03-react/04-converter/src/main.tsx:35]: functions was copied so forgot to change the variable😊

**Pull request merged**